FRE/fre_make.json

```diff
@@ -0,0 +1,202 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+        "name": {
+            "description": "The name of the experiment",
+            "type": "string"
+        },
+        "platform": {
+            "description": "The platforms listed in the command",
+            "type": "string"
+        },
+        "target": {
+            "description": "The targets listed in the command",
+            "type": "string"
+        },
+        "build": {
+           "type": "object",
+           "additionalProperties": false,
+           "properties": {
+               "compileYaml": {
+                   "description": "Path to the compile yaml.",
+                   "type": "string"
+               },
+               "platformYaml": {
+                   "description": "Path to the platform yaml.",
+                   "type": "string"
+               }
+           }
+        },
+        "compile": {
+            "description": "The source code descriptions",
+            "$ref": "#/definitions/Compile"
+        },
+        "platforms": {
+            "description": "FRE platforms",
+            "type": "array",
+            "items": {"$ref": "#/definitions/Platform"}
+        }
+    },
+    "definitions": {  
```

Comment:
I believe this section should be called `$defs` according to the documentation on json-schema.org:

https://json-schema.org/understanding-json-schema/structuring

Reply:
Would my references have to change too?
So `"$ref": "#/definitions/Compile"` --> `"$ref": "#/defs/Compile"

Reply:
Yep, I believe they would change to `"$ref": "#/$defs/Compile"`.